### PR TITLE
PS-269: Re-record innodb.partition, partition_myisam, percona_doublewrite (8.0)

### PR DESCRIPTION
**Pull request by inikep**
1. Add suppression to `innodb.partition`
2. Re-record `innodb.partition`, `main.partition_myisam`, `innodb.percona_doublewrite`

### CODE REVIEW

File: mysql-test/r/partition_myisam.result [hunk 1]
```diff
@@ -79,10 +79,10 @@ a	b
 54	54
 SHOW STATUS WHERE Variable_name LIKE 'Handler%' AND Value >0;
 Variable_name	Value
-Handler_commit	3
-Handler_external_lock	6
+Handler_commit	1
```
Comment (laurynas-biveinis) [inline mysql-test/r/partition_myisam.result:82]: Why there is this difference?

File: mysql-test/suite/innodb/r/partition.result [hunk 2]
```diff
@@ -437,6 +437,7 @@ DROP TABLE t1;
 #
 # Bug#21957001 INNODB: FAILING ASSERTION: 0 IN FILE HA_INNOPART.CC LINE 3526
 #
+call mtr.add_suppression("\\[Warning\\] .*MY-\\d+.* Cannot save statistics for table `test`\\.`t1`");
```
Comment (laurynas-biveinis) [inline mysql-test/suite/innodb/r/partition.result:440]: Is this suppression missing for upstream too?

Comment (inikep) [reply mysql-test/suite/innodb/r/partition.result:440]: No, `Cannot save statistics` is Percona warning

File: mysql-test/r/partition_myisam.result (hunk 1 above)
Comment (inikep) [reply mysql-test/r/partition_myisam.result:82]: I'm almost sure it's https://bugs.mysql.com/bug.php?id=93430

Comment (inikep) [reply mysql-test/r/partition_myisam.result:82]: It's 93430. I verified it with MySQL 8.0.13.

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**